src/main/scala/beam/sim/BeamServices.scala

```diff
@@ -29,4 +31,6 @@ case class BeamServices @Inject()(protected val injector: Injector) extends Acto
   var schedulerRef: ActorRef =_
   var taxiManager: ActorRef = _
   var popMap: Option[Map[Id[Person], Person]] = None
```

Comment:
please refactor popMap to "persons" and remove the Option wrapper to make consistent with vehicles and households.